pkg/crc/machine/delete.go

```diff
@@ -28,10 +28,10 @@ func (client *client) Delete() error {
 
 	// Remove the podman system connection for crc
 	if err := podman.RemoveRootlessSystemConnection(); err != nil {
-		return err
+		logging.Debugf("Failed to remove podman rootless system connection: %v", err)
 	}
 	if err := podman.RemoveRootfulSystemConnection(); err != nil {
-		return err
+		logging.Debugf("Failed to remove podman rootful system connection: %v", err)
```

Comment:
but now we are hiding errors in the podman preset case.

Reply:
@cfergeau yes, we shouldn't block delete operation, if you look https://github.com/code-ready/crc/blob/main/pkg/crc/machine/delete.go#L37-L41 we also ignore here but warn it, do you think warn is better than `Debug`, I can change that also?

Reply:
The ideal situation would be:
```
func (client *podmanClient) Delete() {
    commonDelete()
    if err := podman.RemoveRootfulSystemConnection(); err != nil {
        ....
    }
}

func (client *openshiftClient) Delete() {
    commonDelete()
    if err := cleanKubeconfig(getGlobalKubeConfigPath(), getGlobalKubeConfigPath()); err != nil {
		if !errors.Is(err, os.ErrNotExist) {
			logging.Warnf("Failed to remove crc contexts from kubeconfig: %v", err)
		}
	}
}
```

This way we never run unneeded code. But we are far from that at the moment.